lib/stanford-mods/searchworks.rb

```diff
@@ -37,9 +37,8 @@ def sw_language_facet
                     result << SEARCHWORKS_LANGUAGES[v.strip]
                   end
                 end
-              rescue
-                # TODO:  this should be written to a logger
-                p "Couldn't find english name for #{ct.text}"
+              rescue RuntimeError
```

Comment:
yes, i just removed the exception, deal with the input scenario by checking for it and returning nothing (which would have happened before), but kept the logging

added tests for this